src/js/App/Sidenav/Navigation.scss

```diff
@@ -1,5 +1,5 @@
-.ins-c-sidebar#ins-c-sidebar .pf-c-nav .pf-c-nav__list .pf-c-nav__link {
-    &.ins-c-navigation__additional-links {
+.pf-c-page__sidebar#ins-c-sidebar .pf-c-page__sidebar-body .pf-c-nav .pf-c-nav__list .pf-c-nav__item {
```

Comment:
Can ve make it not to use pf selector? Maybe use some wrapping component selector or add some if it's missing.

Reply:
Yep, actually i think it is safe to just use the class name. It is our class and it is used only at one place.